### PR TITLE
[wip] testing after prow update

### PR DESCRIPTION
**Pull request by rmohr**

### CODE REVIEW

File: hack/builder/publish.sh [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
-docker tag kubevirt/builder:${VERSION} docker.io/kubevirt/builder:${VERSION}
-docker push docker.io/kubevirt/builder:${VERSION}
+cleanup
+
+for ARCH in ${ARCHITECTURES}; do
+    docker push kubevirt/builder:${VERSION}-${ARCH}
+    TMP_IMAGES="${TMP_IMAGES} kubevirt/builder:${VERSION}-${ARCH}"
+done
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
```
Comment (enp0s3) [inline hack/builder/publish.sh]: @rmohr Which experimental feature are we using?

Comment (rmohr) [reply hack/builder/publish.sh]: Just pushed something for testing prow. These changes are actually part of #3045. Building multi-arch manifests is an experimental feature on the docker client.

**Comment by rmohr** [comment]
/close

prow behaviour looks good.

**Pull request closed without merging**